backend/src/Controller/ProductController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\ProductRepository;
+use App\Entity\Product;
+
+class ProductController extends AbstractController
+{
+    /**
+     * @Route("/products", methods={"GET"})
+     */
+    public function index(
+        ProductRepository $repository, 
+        SerializerInterface $serializer): JsonResponse
```

Comment:
Maybe it would be helpful to write your own serializer, so that you won't have any conflicts with the entity-relations?

Reply:
Yes, I will keep this in mind. But at the moment I do not need any entity relations